google-cloud-video-stitcher/samples/Gemfile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 Google, Inc
```

Comment:
Use `Google LLC` rather than `Google, Inc`. Likewise for several of the other files.

Reply:
Done